src/evtx_record.rs

```diff
@@ -34,6 +35,13 @@ pub struct SerializedEvtxRecord {
     pub data: String,
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub struct EvtxRecordWithJsonValue {
+    pub event_record_id: u64,
+    pub timestamp: DateTime<Utc>,
+    pub data: Value,
```

Comment:
There seems to be some duplication between this and `SerializedEvtxRecord`.

Perhaps there is a common trait between the output targets?
So some trait that `EvtxOutput` can store results into?